base/experiment.go

```diff
@@ -76,6 +78,9 @@ type Insights struct {
 	// NumVersions is the number of app versions detected by Iter8
 	NumVersions int `json:"numVersions" yaml:"numVersions"`
 
+	// VersionNames is list of version identifiers if known
```

Comment:
```go
VersionNames []VersionInfo

type VersionInfo struct {
  Version string
  Track  string
}
```

The above is the preferred data structure. This way you would be able to output names like this -- `baseline (x9183)` and `candidate (y2849)`. The things in the parentheses are the version names, which might often be unintelligible even to the developer/ops person (because the version names might be based off of commit IDs, COS buckets, etc.).

Reply:
done